### ISSUE TITLE
Angular 15 Update Peer Dependencies

### ISSUE THREAD

**Issue by verifiedengineer** [Maxl94/ngx-multi-sort-table#106]
First of all, thank you for your package and work!

May I kindly ask you to update your npm package to Angular 15 since your package is the only one not yet supported in my package list.

**Comment by Maxl94** [comment]
HI, I plan to update to angular 15 around the Christmastime.

**Comment by aikrez** [comment]
Hello, any update?

**Comment by Maxl94** [comment]
Hi together, since the update the `checkboxes` are not rendered correctly anymore. At the moment, I unfortunately have no idea what this could be due to. 

If anyone has a suggestion or even a fix, this would be really welcome. The code can be found [here](https://github.com/Maxl94/ngx-multi-sort-table/tree/106-angular-15-update-peer-dependencies).

I don't work as a web developer anymore and therefore don't have much to do with Angular.

**Comment by cnebe** [comment]
While i cannot promise anything i ll take a look into it during the next days as this is a highly wanted future for our project.

**Comment by cnebe** [comment]
@Maxl94 you simply need to replace 
```
import { MatLegacyCheckboxModule as MatCheckboxModule } from '@angular/material/legacy-checkbox';
```
by the newer version without any change required. It looks like the legacy version is not compatible to the newer animation module. The code snippet needs to be replaced inside of the demonstration app and in the module itself.

If you whish i could do some more cleanup, but ill require some permission on that project for that.

**Comment by Maxl94** [comment]
@cnebe thanks for your work. I will fix this now. 

For the cleanup, cant you just create  a PR?

**Comment by Maxl94** [comment]
This issue is closed, since the update to angular 15 is released with `0.10.0`.
